test/e2e/testing-manifests/storage-csi/gce-pd/csi-controller-rbac.yaml

```diff
@@ -107,11 +118,49 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: csi-gce-pd-controller-sa
+    namespace: default
 roleRef:
   kind: ClusterRole
   name: csi-gce-pd-resizer-role
   apiGroup: rbac.authorization.k8s.io
 
+---
+# xref: https://github.com/kubernetes-csi/external-snapshotter/blob/master/deploy/kubernetes/csi-snapshotter/rbac-csi-snapshotter.yaml
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: csi-gce-pd-snapshotter-role
+rules:
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get", "list"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshotclasses"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshotcontents"]
+    verbs: ["create", "get", "list", "watch", "update", "delete"]
```

Comment:
do we need create/update permission? I thought snapshot controller is supposed to do that?  Also who is supposed to handle delete? cc @xing-yang

Reply:
Also add patch. I think patch is preferred over update

Reply:
Actually we should not need "create" as the common snapshot controller will be creating the content.
Sidecar is responsible for updating finalizers, content status, and content deletion so we do need "update" and "delete" here.